tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -464,3 +464,10 @@ platform_tests/api/test_thermal.py::TestThermalApi::test_set_high_threshold:
     reason: "Unsupported platform API"
     conditions:
       - "asic_type in ['mellanox']"
+
+ntp/test_ntp.py::test_ntp_long_jump_disabled:
```

Comment:
Sorry, one last thing, please move this condition above according to alphabetic order of test cases.

Reply:
fixed